appveyor.yml

```diff
@@ -2,34 +2,67 @@ platform: x64
 configuration: Release
 image: 
  - Visual Studio 2017
- - Visual Studio 2013
+#  - Visual Studio 2013
+
+# Do we want to have a Matrix? 32-bit and 64-bit?
```

Comment:
64bit only should be enough. We only provide a 64bit SDK and nobody complained so far